runtime-light/tl/tl.cmake

```diff
@@ -1,4 +1,2 @@
-prepend(RUNTIME_TL_SRC tl/
-        tl-builtins.cpp
-        tl-core.cpp
-)
+prepend(RUNTIME_TL_SRC tl/ tl-builtins.cpp tl-core.cpp tl-functions.cpp
+        tl-types.cpp)
```

Comment:
Here you use tab before tl-types.cpp, but in `runtime-light/stdlib/stdlib.cmake` don't

Reply:
Actually, there is not `tab` symbol, there are space symbols. But again, this is just a formatting used in my editor. I think we need to come up with a common decision for formatting of `cmake` files.